### DISCUSSION TITLE
Problem to assign IP address to interface

### DISCUSSION THREAD

**Discussion by PeterKnotek** [netbox-community/netbox#17506] [General]
HI

After update to latest version have problems with add IP to interface 
- no problem create IP
- no problem search device
  - problem search interfaces assigned to devices
    - not show any interface ( physic or virtual )
    - ![obrázok](https://github.com/user-attachments/assets/e973927a-cc0f-4085-bac6-ad89bf3328c9)

And in devices can click to interface where asigned to device.
![obrázok](https://github.com/user-attachments/assets/417ef480-a0f8-4f9d-bb18-240b7272d7ab)

I'm an administrator and I have no problems with the latest update

Thanks

**Comment by PeterKnotek** [comment]
Problem solved

It was allowed
Admin // User Preferences // HTMX Navigation // Enabled
After turning off this function - everything works again

![obrázok](https://github.com/user-attachments/assets/755ab91e-7599-4008-9113-b50292bee580)


Thanks

**Comment by ghost** [comment]
https://github.com/netbox-community/netbox/issues/17313
